core/Network.js

```diff
@@ -82,11 +81,11 @@ Network.prototype.run = function (runtime, options, callback) {
     port[1].setRuntime(runtime);
   }
 
-  this._processes.forEach(function (process) {
+  _.forEach(this._processes, function (process) {
     process.inports.forEach(setPortRuntime);
     process.outports.forEach(setPortRuntime);
   });
-  runtime.run(this._processes, options, callback || function () {
+  runtime.run(_.values(this._processes), options, callback || function () {
```

Comment:
`runtime.run` takes a simple array of processes, so we need to extract them from the assoc array. This does that.